documentation/src/main/asciidoc/topics/con_shared_cache_stores.adoc

```diff
@@ -0,0 +1,37 @@
+[id='shared-cache-stores_{context}']
+= Shared cache stores
+
+{brandname} cache stores can be local to a given node or shared across all nodes in the cluster.
+
+Configuring cache stores as shared, `shared`, or local only, `shared(false)`,  impacts the rest of your persistent storage configuration.
+
+[IMPORTANT]
+====
+Never configure shared cache stores to fetch state or purge on startup.
+This configuration can result in data loss and performance issues.
+You should fetch state and purge on startup only if the cache store is local, `shared(false)`.
```

Comment:
Purge on startup is the preferred method for use with a non shared store in a cluster due to loading stale entries otherwise. I know this was documenting somewhere at some point.

Reply:
Well now it's documented here. :wind_chime: